akka-pi-os.yml

```diff
@@ -75,6 +75,17 @@ write_files:
     permissions: '0755'
     path: /usr/local/bin/install-java
 
+  - content: |
```

Comment:
@agolubev It might make sense to create a separate `cloud-init` yml file for this set-up. Like `akka-pi-display-os.yml`...
WDYT?

Reply:
Well, it's very easy to transform led cluster to oled one if you have panel assembled. I can definitely create `pi-display` one, but as soon as we have one set of exercises and pretty much compatible I would stay with single init.yml